pimd/pim_mlag.c

```diff
@@ -552,6 +552,18 @@ static inline void pim_mlag_vxlan_state_update(void)
 
 
 /********************API to process PIM MLAG Data ************************/
+static void pim_mlag_peer_zebra_flag_set(void)
```

Comment:
The naming is ambiguous... It says flag set, but it's not clear what flag. IMO the naming should be clear.

Reply:
@ton31337 we are setting the flags 
`SET_FLAG(router->mlag_flags, PIM_MLAGF_PEER_ZEBRA_UP);`